releasenotes/notes/bit-classes-register-index-optional-9aa6feff4fce3524.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    The constructors of the :class:`~qiskit.circuit.Bit` class and subclasses,
+    :class:`~qiskit.circuit.Qubit`, :class:`~qiskit.circuit.Clbit`, and
+    :class:`~qiskit.circuit.AncillaQubit`, have been updated such that their
+    two parameters, ``register`` and ``index`` are now optional.
```

Comment:
Maybe have an example here? But, we can just do it in the release time roundup.

Reply:
Good point, I'll make sure to add it prior to release.